core/trino-main/src/main/java/io/trino/operator/scalar/DateTimeFunctions.java

```diff
@@ -85,6 +89,7 @@
     private static final int MILLISECONDS_IN_HOUR = 60 * MILLISECONDS_IN_MINUTE;
     private static final int MILLISECONDS_IN_DAY = 24 * MILLISECONDS_IN_HOUR;
     private static final int PIVOT_YEAR = 2020; // yy = 70 will correspond to 1970 but 69 to 2069
+    private static final Slice ISO_8861_DATE_FORMAT = Slices.utf8Slice("%Y-%m-%d");
```

Comment:
Did you mean ISO 8601 ?

Reply:
ups.. thanks!

https://github.com/trinodb/trino/pull/15756